views/helpers.go

```diff
@@ -16,37 +25,47 @@ type (
 		Callback string
 		// User is the stored logged-in user
 		User user.User
+		// Version is the version that is running
+		Version string
 	}
 )
 
 func (v *Views) getData(s *sessions.Session) *Context {
 	val := s.Values["user"]
-	u := user.User{}
+	var u user.User
 	u, ok := val.(user.User)
 	if !ok {
 		u = user.User{Authenticated: false}
 	}
 	c := Context{
 		Callback: "/internal",
 		User:     u,
+		Version:  v.conf.Version,
 	}
 	return &c
 }
 
 // DBToTemplateType converts from the DB layer type to the user template type
-func DBToTemplateType(dbUser *[]user.User) []UserStripped {
-	var tplUsers []UserStripped
-	for i := range *dbUser {
-		user1 := UserStripped{}
-		user1.UserID = (*dbUser)[i].UserID
-		user1.Username = (*dbUser)[i].Username
-		user1.Nickname = (*dbUser)[i].Nickname
-		user1.Name = (*dbUser)[i].Firstname + " " + (*dbUser)[i].Lastname
-		user1.Email = (*dbUser)[i].Email
-		user1.Avatar = (*dbUser)[i].Avatar
-		user1.UseGravatar = (*dbUser)[i].UseGravatar
-		if (*dbUser)[i].LastLogin.Valid {
-			user1.LastLogin = (*dbUser)[i].LastLogin.Time.Format("2006-01-02 15:04:05")
+func DBToTemplateType(dbUsers []user.User) []user.StrippedUser {
+	var tplUsers []user.StrippedUser
+	for _, dbUser := range dbUsers {
+		var user1 user.StrippedUser
```

Comment:
Could this variable name be more meaningful?

Reply:
I've changed this now